torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -150,7 +150,7 @@ RegisterOperators reg(
          },
          aliasAnalysisConservative()),
      Operator(
-         "aten::backward(Tensor(a!) self, Tensor? gradient=None, bool? retain_graph=None, bool create_graph=False) -> ()",
+         "aten::backward(Tensor self, Tensor? gradient=None, bool? retain_graph=None, bool create_graph=False) -> ()",
```

Comment:
I guess we could try to delete this overload here, as in VariableTypeManual we already this as an operator.